wemake_python_styleguide/visitors/tokenize/primitives.py

```diff
@@ -221,6 +224,15 @@ def _check_wrong_unicode_escape(self, token: tokenize.TokenInfo) -> None:
             # character can never be the start of a new backslash escape.
             index += 2
 
+    def _check_unnecessary_raw_string(self, token: tokenize.TokenInfo) -> None:
+        modifiers, string_def = split_prefixes(token)
+
+        if 'r' in modifiers.lower():
+            if '\\' not in string_def.encode('unicode-escape').decode():
+                self.add_violation(
+                    RawStringNotNeededViolation(token, text=token.string),
```

Comment:
`text=` can be removed.

Reply:
See c6825cb4e8c8f27480c5fe98bba4ebbea0546b16